adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -180,16 +180,17 @@ protected void validate(Schema inputSchema) {
           String fieldName = outputField.getName();
           Schema.Field inputField = inputSchema.getField(outputField.getName());
           if (inputField == null) {
-            throw new IllegalArgumentException("Input record does not contain the " + fieldName
-                                                 + " field in output schema.");
+            throw new IllegalArgumentException("Input record does not contain the [" + fieldName
```

Comment:
I didn't mean to use literal brackets, but was pointing out the error message is wrong. It should be about the input schema, as there are no input records at configure time.